FluentValidation.AutoValidation.Mvc/src/Filters/FluentValidationAutoValidationActionFilter.cs

```diff
@@ -55,7 +55,7 @@ public async Task OnActionExecutionAsync(ActionExecutingContext actionExecutingC
                     if (actionExecutingContext.ActionArguments.TryGetValue(parameter.Name, out var subject))
                     {
                         var parameterInfo = (parameter as ControllerParameterDescriptor)?.ParameterInfo;
-                        var parameterType = parameter.ParameterType;
+                        var parameterType = subject!.GetType();
```

Comment:
Do we know for sure that `subject` is not null if the TryGetValue returns true in this case? Further down there is a null check on `subject`.

Reply:
I just pushed another commit to support a potentially nullable `subject`.

Thank you.